execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -277,6 +281,26 @@ impl ExecConfig {
     pub fn push_account(&mut self, account: GenesisAccount) {
         self.accounts.push(account)
     }
+
+    pub fn validator_slots(&self) -> u32 {
+        self.validator_slots
+    }
+}
+
+impl fmt::Debug for ExecConfig {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
+        write!(
+            f,
+            "validator_slots: {}, mint_installer_bytes: {:10}, proof_of_stake_installer_bytes: {:10}, standard_payment_installer_bytes: {:10}, auction_installer_bytes: {:10}, wasm_costs: {:?}, accounts: {:?}",
+            &self.validator_slots,
+            HexFmt(&self.mint_installer_bytes),
+            HexFmt(&self.proof_of_stake_installer_bytes),
+            HexFmt(&self.standard_payment_installer_bytes),
+            HexFmt(&self.auction_installer_bytes),
+            &self.wasm_costs,
+            &self.accounts,
+        )
```

Comment:
This would fit well `Display` impl, but for `Debug` we're trying to provide as verbose information as possible. Something like:

```suggestion
        f
        .debug_struct("ExecConfig")
        .field("validator_slots", &self.validator_slots)
        .field("mint_installer_bytes", HexFmt(&self.mint_installer_bytes[..5]))
        // ...
        .finish()
```
```

Reply:
Actually we use HexFmt in many Debug implementations. But at any rate this entire impl doesn't need to be in this PR; I added it while I was debugging the logic and left it in as it seemed useful. However it isn't worth slowing this PR down over, so I just reverted to deriving Debug on Exec_Config.

Reply:
It's a shame this didn't get merged as it was.  Derived Debug impls containing Wasm are horrible!

I disagree with @mpapierski suggestion of `HexFmt(&self.mint_installer_bytes[..5])` - we should either be using the style @EdHastingsCasperLabs originally committed, the full hex string (inappropriate here due to the size of that string potentially being massive) or what we have also used in some places is just an indication of the number of bytes in the field where the actual contents are probably not that useful (which is usually the case for raw Wasm I'd think).

See for example [`ExecutableDeployItem`](https://github.com/CasperLabs/casper-node/blob/master/execution_engine/src/core/engine_state/executable_deploy_item.rs#L311-L432) where we have a mix of all three styles.  I don't think we should be using any other styles for formatting byte strings than these three, for the sake of consistency.